gulpfile.js

```diff
@@ -66,18 +68,41 @@ gulp.task('show-language-plugin', function (cb) {
 						if (title !== ucfirst) {
 							languagesMap[p] = title;
 						}
+
+						if(data.languages[p].require) {
+							dependenciesMap[p] = data.languages[p].require;
+						}
 					}
 				}
 
-				var jsonLanguages = JSON.stringify(languagesMap);
-				var stream = gulp.src(paths.showLanguagePlugin)
-					.pipe(replace(
-						/\/\*languages_placeholder\[\*\/[\s\S]*?\/\*\]\*\//,
-						'/*languages_placeholder[*/' + jsonLanguages + '/*]*/'
-					))
-					.pipe(gulp.dest(paths.showLanguagePlugin.substring(0, paths.showLanguagePlugin.lastIndexOf('/'))));
-				stream.on('error', cb);
-				stream.on('end', cb);
+				var jsonLanguagesMap = JSON.stringify(languagesMap);
+				var jsonDependenciesMap = JSON.stringify(dependenciesMap);
+
+				var tasks = [
+					{plugin: paths.showLanguagePlugin, map: jsonLanguagesMap},
+					{plugin: paths.autoloaderPlugin, map: jsonDependenciesMap}
+				];
+
+				var cpt = 0;
+				var l = tasks.length;
+				var done = function() {
+					cpt++;
+					if(cpt === l) {
+						cb && cb();
+					}
+				};
+
+				tasks.forEach(function(task) {
+					var stream = gulp.src(task.plugin)
+						.pipe(replace(
+							/\/\*languages_placeholder\[\*\/[\s\S]*?\/\*\]\*\//,
+							'/*languages_placeholder[*/' + task.map + '/*]*/'
```

Comment:
Yeah, I was just wondering if the comment needed to be included in the final product. It's not a huge deal :).